suite-native/module-authorize-device/src/screens/thp/ThpConfirmationScreen.tsx

```diff
@@ -21,19 +23,33 @@ export const ThpConfirmationScreen = ({
     navigation: NativeStackNavigationProp<AuthorizeDeviceStackParamList>;
 }) => {
     const navigateToInitialScreen = useNavigateToInitialScreen();
-    const { showThpAutoconnectEnableAlert } = useThpAutoconnectAlerts();
+
+    const { startAutoconnect, ignoreAutoconnect } = useThpAutoconnectActions();
 
     const thpStep = useSelector(selectThpStep);
 
+    const { showAlert } = useAlert();
+
+    const handleShowThpAutoconnectEnableAlert = useCallback(() => {
```

Comment:
Can we keep the name `showThpAutoconnectEnableAlert`?

Reply:
I renamed to `handleXXX` to keep it consistent with most handlers inside components across the repo. Why do you prefer the previous name? Just because it's more concise?

Reply:
Because it's not handling any event or something similar. It does show an alert.

Reply:
`showAlert` is not called `handleShowAlert` either.

Reply:
Yes, but most handlers inside components have a handle prefix. We used to have a naming convention to name handlers with this prefix and if it is passed as a prop, the prop name of children uses `onHandle` prefix. However I don't have a strong opinion on this and reverted the name, I think it doesn't make much difference. It's rebased already, but name is changed.